### PR TITLE
Add files : index.html, style.css and .gitignore

### PR DESCRIPTION
**Pull request by sarmade**

### CODE REVIEW

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @sarmade , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders

## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to add or improve
  - [x] Please make sure to have a professional README file. This is a general requirement. I will highly recommend following the [microverse template](https://github.com/microverseinc/readme-template)
  
  - [x] Please make sure Linter is set up correctly for this repository and all checks are green. You can find the linter template [here](https://github.com/microverseinc/linters-config)
  
  - [x] As part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO
  
  - [x] Your Pull Request should have a summary describing introduced changes in more detail (you can edit it at the top of the PR)

_I'm sure you can do it just keep it up_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Hello</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <h1>Hello Microverse!</h1>
+</body>
```
Comment (m05e5) [inline index.html]: As part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO

**Comment by sarmade** [comment]
@m05e5
When I make the check button, github told me :
Workflow runs completed with no job.

That mean that is ok or not ?

**Comment by sarmade** [comment]
@m05e5
I have created by mistake 5 pull requests.
This is an error or not?

**Comment by sarmade** [comment]
@m05e5
You should use this link now: https://github.com/sarmade/Hello/pull/7

**Pull request merged**